thirdparty/CMakeLists.txt

```diff
@@ -409,3 +409,11 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+ExternalProject_Add(abseil
+        URL https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.zip
```

Comment:
could U add one more url?

`https://pegasus-thirdparty-package.oss-cn-beijing.aliyuncs.com/abseil-20230802.1.zip.zip`

Reply:
Be like ` ${OSS_URL_PREFIX}/abseil-20230802.1.zip`